klaus/views.py

```diff
@@ -274,16 +274,15 @@ def make_template_context(self, *args):
             (readme_filename, readme_data) = self._get_readme()
         except KeyError:
             self.context.update({
-                'is_markup': None,
+                'is_markup': False,
```

Comment:
Why this change?

Reply:
It was an experiment before I found the fix. So the change is not necessary but I left it there because `is_markup` sounds like a boolean (what would it mean if it was None?). But AFAIK None and False are equivalient in boolean expressions. You decide.